packages/perseus/src/index.ts

```diff
@@ -109,11 +109,28 @@ export {
     getImagesWithoutAltData,
 } from "./util/extract-perseus-data";
 export {
+    /**
+     * @deprecated - import this function from perseus-core instead
```

Comment:
My vote would be to make this a major release for `perseus` and remove these from the exports.

Reply:
> Are these the "pre-parsing" types?

Not sure I understand. `parsePerseusItem` is currently used in webapp, and I want to avoid causing any headaches when we do the perseus release this week.